core/api_requests.py

```diff
@@ -50,36 +51,15 @@ def get_history(id):
     response = requests.post(URL, json={'query': query_history(id)})
 
     return response.json()
-
 def get_rockets():
     """
     request rockets information
     """
     response = requests.post(URL, json={'query': query_rockets()})
     return response.json()
-
 def get_rocket(id):
     """
     request single rocket information
     """
     response = requests.post(URL, json={'query': query_rocket(id)})
-    return response.json()
-
-
-def get_missions():
-    """
-    Requests missions data from API
-    """
-
-    response = requests.post(URL, json={'query': query_missions()})
-
-    return response.json()
-
-def get_mission(id):
```

Comment:
## pt

Novamente, por que remover a consulta por missão?

## en

Onde again, why remove the mission?